dynamic_array.py

```
@@ -0,0 +1,113 @@
+    - get: O(1)
+    - set: O(1)
+    - remove: O(n)
+    - size: O(1)
+    - is_empty: O(1)
+    - clear: O(1)
+    
+    Space Complexity:
+    - O(n), where n is the number of elements stored.
+    """
+    
+    def __init__(self, capacity=2):
```

Ruff (0.15.18)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)